pom.xml

```diff
@@ -32,7 +32,7 @@
 	</parent>
 	<groupId>org.eclipse.tractusx</groupId>
 	<artifactId>auto-setup</artifactId>
-	<version>1.4.0</version>
+	<version>1.4.2</version>
 	<name>auto-setup</name>
 	<description>auto-setup</description>
```

Comment:
This shouldalso be "managed-service-orchestrator"

Reply:
Done